GVFS/GVFS.FunctionalTests/Tools/GVFSProcess.cs

```diff
@@ -92,6 +92,18 @@ public string Status(string trace = null)
             return this.CallGVFS("status " + this.enlistmentRoot, trace: trace);
         }
 
+        public string Health(string directory = null)
+        {
+            if (string.IsNullOrEmpty(directory))
+            {
+                return this.CallGVFS("health " + this.enlistmentRoot);
+            }
+            else
+            {
+                return this.CallGVFS("health -d " + directory + " " + this.enlistmentRoot);

```

Comment:
Do any of the paths in these commands need their directories quoted? What if someone has this inside a directory with a space?

Reply:
From what I have seen - and perhaps more to the point, with the directories I chose to use when writing the functional tests, they all have underscores instead of spaces. It is possible that if someone were to edit the functional tests they could point it to a directory that had a space in which case it seems like they would need to be quoted. Is this an issue we could see potentially happening?

Reply:
The issue is that `this.enlistmentRoot` could have a space in it. If `directory` is always local to the test repo, then there will not be space there.

Reply:
Ah alright, that makes sense. I can add in the quotes around `this.enlistmentRoot` to ensure that isn't a problem. Does it make sense to add them around `directory` just in case? Otherwise, I can leave it as is.